src/main/resources/plugin.yml

```diff
@@ -3,6 +3,7 @@ author: Wythe
 version: 2.6.1
 main: de.False.BuildersWand.Main
 softdepend: [ASkyBlock, WorldGuard, Towny, GriefPrevention, Factions, mcMMO]
+api-version: 1.13
```

Comment:
That appearently fixed the #30 issue... It searched for a legacy material, which it could not find, so it just did nothing at all.
Don't get confused, the plugin is still compatible with 1.8-1.14, even though, here stands "1.13"